src/main/java/com/ericsson/eiffel/remrem/generate/cli/CLIOptions.java

```diff
@@ -140,5 +143,18 @@ public static boolean hasParsedOptions() {
         if (commandLine == null)
             return false;
         return commandLine.getOptions().length > 0;
+    }    
+    
+    public static void handleJarLocation(){
+    	if (commandLine.hasOption("jarDirectory")) {
```

Comment:
jarDirectory should be jarPath. This is a general comment. Use only one name. So jarpath everywhere (can be camelcase where applicable) but do not use jarDirectory or jar_locaion or similar.